AnkiDroid/src/main/java/com/ichi2/anki/stats/AnkiStatsTaskHandler.java

```diff
@@ -194,19 +195,19 @@ public DeckPreviewStatistics() {
         }
 
         @Override
-        protected String doInBackground(Object... params) {
+        protected String doInBackground(Pair<Collection, TextView>... params) {
             //make sure only one task of CreateChartTask is running, first to run should get sLock
             //only necessary on lower APIs because after honeycomb only one thread is used for all asynctasks
             sLock.lock();
             try {
-                Collection collection = (Collection) params[0];
+                Collection collection = params[0].first;
                 if (!mIsRunning || collection == null || collection.getDb() == null) {
                     Timber.d("Quitting DeckPreviewStatistics before execution");
                     return null;
                 } else {
                     Timber.d("Starting DeckPreviewStatistics");
                 }
-                mTextView = (TextView) params[1];
+                mTextView = params[1].second;
```

Comment:
This looks incorrect (params only has one element). Could you put in a regression test if this wasn't caught by our suite
 
```suggestion
                mTextView = params[1].second;
```